app/assets/stylesheets/app_configs/_edit.css.scss

```diff
@@ -0,0 +1,6 @@
+#app_config_reset_tos_for_users {
+  margin-top: 11px;
+  width: auto;
```

Comment:
I'm pretty sure this is the only line that you need, everything else seems unnecessary (based on tweaking via Firebug and then editing the file). Let's keep our hacks as minimal as possible :smile: